src/new-components/heroes/heroes.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import { Canvas, Meta } from '@storybook/addon-docs';
+import { AuroraHero } from './aurora-hero'; import { AuroraType } from '../../components/aurora/aurora';
+
+<Meta
+  title="Layout/Heroes" />
+
+# Aurora Hero
```

Comment:
I like to showcase the heroes in a distinct page. Hence this page without any stories but only some examples. We would have two stories (aurora + images) but the image story is useless so I spared the stories at all.